src/server/wesnothd/server.cpp

```diff
@@ -3106,5 +3094,9 @@ int main(int argc, char** argv)
 		}
 	}
 
-	return wesnothd::server(port, keep_alive, config_file, min_threads, max_threads).run();
+#ifdef WIN32
+	SetConsoleTitleA("wesnoth server");
```

Comment:
So you decided not to set the title on Linux…?

Reply:
What would be the point of printing that to cout? That's not a title to anything.

Reply:
I guess you've ultimately decided to scrap this, but… it's a terminal escape code that instructs the terminal to set its own title. That's also why you'd output it directly to cout instead of to one of the log streams.